dali/ft/daftformat.ipp

```diff
@@ -502,12 +508,12 @@ public:
     {
         if (rowStart <= headerLength)
             return 0;
-        return rowStart - headerLength - 1;
+        return rowStart - headerLength;
```

Comment:
Same change to the offset to be compatible with how index virtual file positions are set.  Better to make a small change in spraying then to complicate indexing.